api/v1alpha1/backupstorage_types.go

```diff
@@ -45,10 +45,14 @@ type BackupStorageSpec struct {
 	Description string `json:"description,omitempty"`
 	// CredentialsSecretName is the name of the secret with credentials.
 	CredentialsSecretName string `json:"credentialsSecretName"`
+	// TargetNamespaces is the list of namespaces where the operator will copy secrets provided in the CredentialsSecretsName.
+	TargetNamespaces []string `json:"targetNamespaces,omitempty"`
```

Comment:
This needs to be discussed with Peter and Nuna, giving the user this responsibility wasn't part of the product requirements.

Reply:
Agreed on this approach.